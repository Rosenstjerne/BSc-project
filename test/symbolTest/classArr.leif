class Point {
    var int x, y;
}

var Point[] P;

function null print_point(Point p) {
    print(p.x + p.y);
}

P = new Point[1];

P[0] = new Point;
P[0].x = 1;
P[0].y = 4;

print_point(P[0]); # Should print 5