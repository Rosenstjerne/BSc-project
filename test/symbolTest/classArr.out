
Error in phase Type Checking, line 17:
    Argument 0 in call to function print_point is of type 'unknown', but must be of type 'Point'
